### ISSUE TITLE
Feedback needed when file has finished uploading and processing

### ISSUE THREAD

**Issue by SusanBrown** [Thaerious/NERVE#94]
Either a progress bar or a confirmation of completion would be good.

**Comment by abramoffcwrc** [comment]
I completely agree with this. I think it would be more intuitive to have it load right into the window you're in so you don't have to search for it in the document list after uploading it. Otherwise there should be a confirmation or progress bar, although that seems more clunky.

**Comment by cutic** [comment]
I absolutely agree.
